CRM/Core/Config/MagicMerge.php

```diff
@@ -222,7 +222,7 @@ public function __get($k) {
     if (!isset($this->map[$k])) {
       throw new \CRM_Core_Exception("Cannot read unrecognized property CRM_Core_Config::\${$k}.");
     }
-    if (isset($this->cache[$k])) {
+    if (array_key_exists($k, $this->cache)) {
```

Comment:
Thanks @totten . I guess I never quite realized that isset returns true with an empty string and with FALSE, yet it returns false with NULL. I'll give it a shot.  I did a quick scan and found that there are several tests in the code that use `if (!empty($config->geocodeMethod))` which should work just as well with NULL or '' - but nothing that seems to expect NULL.